CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [2.32.4](https://github.com/mParticle/mparticle-web-sdk/compare/v2.32.3...v2.32.4) (2025-02-10)
```

Comment:
I think this was committed accidentally.

Reply:
looks like this is because i didn't rebase the feature branch properly.  this should be good now